rules/large-file.js

```diff
@@ -0,0 +1,69 @@
+const nodefs = require('fs')
+// eslint-disable-next-line no-unused-vars
+const Result = require('../lib/result')
+// eslint-disable-next-line no-unused-vars
+const path = require('path')
+/**
+ * Check if a list of files is larger than provided file system size.
```

Comment:
Do you mean "size on the file system"?

Reply:
Also, I think this would be larger than or equal.

Reply:
Updated PR.